tendrl/gluster_integration/objects/brick/atoms/create/__init__.py

```diff
@@ -1,11 +1,10 @@
 from tendrl.commons.event import Event
 from tendrl.commons.message import Message
 from tendrl.commons import objects
-from tendrl.gluster_integration.objects.gluster_brick import GlusterBrick
+from tendrl.gluster_integration.objects.brick import Brick
```

Comment:
Please dont mix up this PR and https://github.com/Tendrl/gluster-integration/pull/282 , the latter one requires UI work completion and the current PR needs to be merged without PR 282